s3/replication/common/tests/system/test_head_object.py

```diff
@@ -50,10 +50,16 @@ async def main():
     bucket_name = config.source_bucket_name
     request_id = "dummy-request-id"
 
+    # For Cortx, Default value of version id=None
```

Comment:
comments can be removed from here now, as we have adopted comments before 
version_id = None

Reply:
done